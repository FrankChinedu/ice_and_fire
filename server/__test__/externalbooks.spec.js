const supertest = require('supertest');
const { app } = require('../app');

const server = () => supertest(app);

describe('EXTERNAL BOOKS', ()=>{
  it('should get external books api', async ()=>{
    const { status, body } = await server().get(
      '/api/external-books');
      expect(status).toBe(200);
      expect(Object.keys(body)).toMatchSnapshot();
  });
});
