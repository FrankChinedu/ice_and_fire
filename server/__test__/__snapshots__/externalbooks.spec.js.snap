// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EXTERNAL BOOKS should get external books api 1`] = `
Array [
  "status_code",
  "status",
  "data",
]
`;
